app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,68 @@
+class CommentsController < ApplicationController
+  load_and_authorize_resource :issue
+  load_and_authorize_resource :project
+  load_and_authorize_resource through: %i[issue project]
+
+  # POST /resources/:resource_id/comments
+  def create
+    set_up_pagy_and_resource
```

Comment:
move `set_up_pagy_and_resource` to `before_action`, reason: a method must be written as before action if it used in multiple actions of the same controller.

Reply:
ok

Reply:
done